Dockerfile

```diff
@@ -174,7 +176,7 @@ RUN bundle exec rake assets:precompile
 ADD /server /opt/openstudio/server
 ADD .rubocop.yml /opt/openstudio/.rubocop.yml
 # Run bundle again, because if the user has a local Gemfile.lock it will have been overriden
-RUN bundle install
+RUN bundle update
```

Comment:
Great catch. This change was derived thanks to the following build error:
```shell
Using rake 11.3.0
Using i18n 0.8.0
Using json 1.8.6
Using minitest 5.10.1
Using thread_safe 0.3.5
Using builder 3.2.3
Using erubis 2.7.0
Using mini_portile2 2.1.0
Using rack 1.6.5
Using mime-types-data 3.2016.0521
Using arel 6.0.4
Using minitar 0.6
Your Gemfile.lock is corrupt. The following gem is missing from the DEPENDENCIES
section: 'minitar-cli'
```
Is there a better way of fixing this?